README.md

```diff
@@ -39,10 +48,58 @@ Or install it yourself as:
     $ gem install ar2dto
 
 ## Usage
-TBD
+
+In the following sections, we explain the API provided by the gem and some basic usage.
 
 ### Setting up your models
-TBD
+
+To use the gem, you need to add `has_dto` to your ActiveRecord models. For example:
+```ruby
+class User < ApplicationRecord
+  has_dto
+end
+```
+
+This will dynamically create a class called `UserDTO` and will add two methods to your ActiveRecord model: `#to_dto` and `.to_dto`.
```

Comment:
```suggestion
This will dynamically create a class called `UserDTO` and will add two methods to your ActiveRecord model: [#to_dto](#to_dto) and [.to_dto](#to_dto-1).
```